### DISCUSSION TITLE
Build and Install Cmake (linux)

### DISCUSSION THREAD

**Discussion by ghost** [mdaus/nitro#428] [Q&A] [answered]
Hello, I am running through the build instructions and when trying to build I am running into an error in /externals/coda-oss/modules/c++/include/TestCase.h where it cannot compare (t==u) and (u==t) due to "integer expressions of different signedness: 'const long unsigned it' and 'const int'.

Anyone know what I can do about this?

**Comment by JDanielSmith** [comment, marked as answer]
This is fixed in the latest release.
